lib/generator/execution.rb

```diff
@@ -99,6 +99,10 @@ def self.logic(hqmf_document, population_index=0, options)
           #{observation_function(custom_functions, population_index)}
         }
         
+        var variables = function() {
+          return executeIfAvailable(hqmfjs.VARIABLES, patient_api);
```

Comment:
Since this is only for when we want a complete rationale, we should probably check whether we want the rationale before calculating the variables.